### ISSUE TITLE
SPSearchTopology: FirstPartitionDirectory issue

### ISSUE THREAD

**Issue by NikCharlebois** [PowerShell/SharePointDsc#925]
#### Details of the scenario you tried and the problem that is occurring
The Get-TargetResource of the SPSearchTopology Resource returns the property FirstPartitionDirectory as a string, however it is supposed to be an array. This causes an issue the moment two folders are specified. We need to change the logic to only return the first item.

#### Verbose logs showing the problem
N/A

#### Suggested solution to the issue
```powershell
$ssiComponents = $enterpriseSearchServiceInstance.Components
if($null -ne $ssiComponents)
{
    if($ssiComponents.Length -gt 1)
    {
        $ssiComponents = $ssiComponents[0]
    }
    $firstPartition = $ssiComponents.IndexLocation[0]
}
```

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
N/A

#### The operating system the target node is running
N/A

#### Version of SharePoint that is used (e.g. SharePoint 2016)
2013

#### Version and build of PowerShell the target node is running
PoSh 4

#### Version of the DSC module that was used ('dev' if using current dev branch)
2.5.0.0

**Comment by ykuijs** [comment]
@NikCharlebois Was this issue fixed in PR #928?

**Comment by NikCharlebois** [comment]
Yes it was. Closing the Issue
